google/resource-snippets/pubsub-v1/test_alternatives.sh

```diff
@@ -28,12 +28,23 @@ terraform plan -var="deployment=d1" -var="project_id=${TF_PROJECT_ID}"
 terraform apply -auto-approve -var="deployment=d1" -var="project_id=${TF_PROJECT_ID}"
 popd
 
+# KCC Configuration
+gcloud config set project $KRM_PROJECT_ID
+cp -R alternatives/krm /tmp/krm_${KRM_PROJECT_ID}
+pushd /tmp/krm_${KRM_PROJECT_ID}
+kubectl apply -f pubsub.yaml
+kubectl  wait --for=condition=Ready PubSubSubscription --all
+popd
+rm -rf krm_${KRM_PROJECT_ID}
+
 
 # Export DM and TF resources for comparison
 gcloud pubsub subscriptions list --filter="labels.goog-dm:d1" --project $DM_PROJECT_ID | sed "s/${DM_PROJECT_ID}/PROJECT/" > /tmp/dm.yaml
 
 gcloud pubsub subscriptions list --filter="labels.goog-dm:d1" --project $TF_PROJECT_ID | sed "s/${TF_PROJECT_ID}/PROJECT/"  > /tmp/tf.yaml
 
+gcloud pubsub subscriptions list --filter="labels.goog-dm:d1" --project $KRM_PROJECT_ID | sed "s/${KRM_PROJECT_ID}/PROJECT/" | sed "s/creationTime: .*/creationTime: TIME/" | sed "/cnrm-lease-.*/d" | sed "/managed-by-cnrm.*/d" \  > /tmp/krm.yaml
```

Comment:
The last `\  ` needs to be removed, otherwise sed won't work. Please make sure these test scripts actually pass